src/components/interaction-designer/block-editors/choices/ChoiceMappingModal.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="choice-mapping-modal">
     <div class="form-group">
-      <button :disabled="block.config.choices.length === 0" class="btn btn-primary btn-sm" @click="showModal">
+      <button
+        :disabled="block.config.choices.length === 0"
+        class="btn btn-primary btn-sm"
+        data-cy="set-choice-options"
```

Comment:
add postfix `--btn` according to our convention https://app.getguru.com/folders/iAnRr8nT/Automated-testing?activeCard=59e5bfdf-a2fb-48bf-a805-cd8083b929aa